From e7592364504f49aab8bf353b4c85fe6d88651ee3 Mon Sep 17 00:00:00 2001
From: Jens Remus <jremus@linux.ibm.com>
Date: Mon, 9 Sep 2024 17:05:28 +0200
Subject: [PATCH 082/108] s390: Align opcodes to lower-case

opcodes/
	* s390-opc.txt (rdp): Change opcode to lower-case.

Signed-off-by: Jens Remus <jremus@linux.ibm.com>
(cherry picked from commit 1afe02759f1569fb647b691d308c95efc2116b23)
---
 opcodes/s390-opc.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/opcodes/s390-opc.txt b/opcodes/s390-opc.txt
index 1182e196059..4610a8fddd7 100644
--- a/opcodes/s390-opc.txt
+++ b/opcodes/s390-opc.txt
@@ -2061,7 +2061,7 @@ e60000000055 vcnf VRR_VV0UU2 "vector fp convert to nnp" arch14 zarch
 
 # Reset-DAT-Protection Facility
 
-b98B rdp RRF_RURR2 "reset dat protection" arch14 zarch optparm
+b98b rdp RRF_RURR2 "reset dat protection" arch14 zarch optparm
 
 # BEAR-Enhancement Facility
 
-- 
2.47.0

